commit 27fe22378da38424102c5292b782cacdd9d7b9e4
Author: Julian Seward <jseward@acm.org>
Date:   Tue Nov 20 12:09:03 2018 +0100

    Add support for Iop_{Sar,Shr}8 on ppc.  --expensive-definedness-checks=yes needs them.

diff --git a/VEX/priv/host_ppc_isel.c b/VEX/priv/host_ppc_isel.c
index 5242176..750cf8d 100644
--- a/VEX/priv/host_ppc_isel.c
+++ b/VEX/priv/host_ppc_isel.c
@@ -1528,7 +1528,6 @@ static HReg iselWordExpr_R_wrk ( ISelEnv* env, const IRExpr* e,
                                            True/*32bit shift*/,
                                            tmp, tmp,    amt));
                r_srcL = tmp;
-               vassert(0); /* AWAITING TEST CASE */
             }
          }
          /* Only 64 expressions need 64bit shifts,
